packages/flutter_tools/test/general.shard/pub_dependencies_project_validator_test.dart

```diff
@@ -0,0 +1,77 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_tools/src/base/file_system.dart';
+import 'package:flutter_tools/src/globals.dart' as globals;
+import 'package:flutter_tools/src/project.dart';
+import 'package:flutter_tools/src/project_validator.dart';
+import 'package:flutter_tools/src/project_validator_result.dart';
+
+import '../src/common.dart';
+import '../src/context.dart';
+
+void main() {
+  late FileSystem fileSystem;
+
+  group('PubDependenciesProjectValidator', () {
+
+    setUp(() {
+      fileSystem = globals.localFileSystem;
+    });
+
+    testWithoutContext('success ', () async {
```

Comment:
out of curiosity, why the trailing space in the test name?

Reply:
That was a type